pyproject.toml

```diff
@@ -19,9 +19,33 @@ dependencies = [
     "iso8601",
 ]
 
+[project.optional-dependencies]
+dev = [
+    "build",
+    "coverage",
+    "ipython",
+    "isort",
+    "pre-commit",
+    "pytest",
+    "ruff",
+    "tox",
+    "twine",
+]
+
 [tool.setuptools.dynamic]
 version = {attr = "pyargus.VERSION"}
 
 [tool.setuptools.packages.find]
 where = ["src"]
 exclude = ["tests*"]
+
+[tool.ruff]
+line-length = 88
+extend-exclude = [
+    ".egg-info",
+]
+output-format = "full"
+
+[tool.ruff.lint]
+select = ["E4", "E7", "E9", "F4", "F5", "F6", "F7", "F82", "F9"]
```

Comment:
This will probably change from F82 to F8, we just haven't had the time to merge it yet in argus-server.

Reply:
changed to F8